### ISSUE TITLE
Add brand colors to HIG

### ISSUE THREAD

**Issue by cassidyjames** [elementary/website#1916]
A convention we've never covered

**Comment by bumper314** [comment]
Does your request go beyond what's already covered in [Color Palette](https://elementary.io/fi/docs/human-interface-guidelines#color)?

I see from `git blame` that most of those colors were added to the HIG in Feb 2018.

**Comment by cassidyjames** [comment]
@bumper314 yes, we encourage developers to use colored headerbars and accent colors, as seen in apps like [Snaptastic](https://appcenter.elementary.io/com.github.bartzaalberg.snaptastic/) or [Planner](https://appcenter.elementary.io/com.github.alainm23.planner/). This is something we support in the stylesheet but have not yet documented in the HIG.
